src/common/UserAvatar.js

```diff
@@ -20,32 +24,36 @@ type Props = $ReadOnly<{|
  * @prop [children] - If provided, will render inside the component body.
  * @prop [onPress] - Event fired on pressing the component.
  */
-export default class UserAvatar extends PureComponent<Props> {
-  render() {
-    const { avatarUrl, children, size, onPress } = this.props;
-    const borderRadius = size / 8;
-    const style = {
-      height: size,
-      width: size,
-      borderRadius,
-    };
+const UserAvatar = function UserAvatar(props: Props) {
+  const { avatarUrl, children, size, onPress } = props;
+  const borderRadius = size / 8;
+  const style = {
+    height: size,
+    width: size,
+    borderRadius,
+  };
 
-    return (
-      <View>
-        <Touchable onPress={onPress} style={style}>
-          <ImageBackground
-            style={style}
-            source={{
-              uri: avatarUrl.get(PixelRatio.getPixelSizeForLayoutSize(size)).toString(),
-            }}
-            resizeMode="cover"
-            /* ImageBackground seems to ignore `style.borderRadius`. */
-            borderRadius={borderRadius}
-          >
-            {children}
-          </ImageBackground>
-        </Touchable>
-      </View>
-    );
-  }
-}
+  const auth = useSelector((state: GlobalState) => getAuth(state));
```

Comment:
This suggests we can probably benefit from a simple type-wrapper for `useSelector`, in our `src/react-redux.js` -- one that just specifies that the callback's argument is of type `GlobalState`.

I can try adding that after this is merged (or while merging it.)

Reply:
Sure, sounds good—thanks!